scripts/step03_explore_examples/02_advanced_rag/main.py

```diff
@@ -36,7 +37,8 @@ def main():
         print("3: Advanced RAG - Self Query")
         print("4: Advanced RAG - Rewrite Retrieve Read")
         print("5: Advanced RAG - RAG Fusion")
-        print("6: Exit\n")
+        print("6: Advanced RAG - HANA Self Query")
```

Comment:
Please align the numbering with: https://github.com/SAP-samples/btp-genai-starter-kit/pull/92.

Reply:
Sure, we will align cc: @sreanikdk